### ISSUE TITLE
~\dub\packages\ae\0.0.2845\ae\utils\iconv.d(111,29): Error: Bidirectional control characters are disallowed for security reasons.

### ISSUE THREAD

**Issue by MagikGimp** [CyberShadow/steamkeyactivator#10]
I get this error message when I try to "dub build" in the directory for your cloned repo. This happens in both Windows and Linux. Am I doing something wrong or is the code too old to be built successfully now? Thanks.

**Comment by CyberShadow** [comment]
Yes, the code is old, but you should still be able to use an older version of D to build it.
